### DISCUSSION TITLE
How does data removal actually work?

### DISCUSSION THREAD

**Discussion by kzshantonu** [umputun/remark42#1753] [Q&A]
Hi, 

So far loving what you've built and thanks so much for this.

I can't seem to figure out how data removals actually work.

A user clicks the removal button, sends me the email. Then what?

Does the user click anything or do I?

I click the link in email while logged in as admin but nothing happens

**Comment by umputun** [comment]
This request has been sent to the admin, I got yours. Clicking on the link does the actual delete

**Reply by umputun** [reply]
not sure what you mean by " logged in as admin " as the one I got was from the demo site

**Reply by kzshantonu** [reply]
That was just a test. I was logged in on my own site and also on my instance's `/web` and nothing was happening

**Reply by umputun** [reply]
Did you set your SMTP settings properly, and do you get any emails? I have not done anything special on the demo to get your notification, it just a typical setup

**Reply by kzshantonu** [reply]
SMTP not configured. Is that needed? I thought the user is supposed to send the email using their own email client, then I as an admin is supposed to click the link. Is that not correct? Am I supposed to click the link or is the user supposed to?

**Reply by umputun** [reply]
Good point. It has been a while since we added removal request functionality, so you are probably correct, and It should send an email with a mailto link. As you (admin) get it, you can click, which will invoke the removal. What part of this flow doesn't work for you?

 @paskal, this email area is something you touched recently. Do you recall how all of this is working, and can you provide any inside?

**Reply by kzshantonu** [reply]
> What part of this flow doesn't work for you?

I click the link as admin but nothing happens

**Reply by kzshantonu** [reply]
Also the demo privacy policy says the user have to click something. Which is why I'm confused
